ale/base/type_sensor.py

```diff
@@ -51,7 +51,7 @@ def ephemeris_time(self):
         : ndarray
           ephemeris times split based on image lines
         """
-        return np.linspace(self.ephemeris_start_time,  self.ephemeris_stop_time, self.image_lines / 64)
+        return np.linspace(self.ephemeris_start_time,  self.ephemeris_stop_time, int(self.image_lines / 64))
```

Comment:
I know this was here before, but in my tests, any image_lines less than 128 only gets one value back from linespace. Larry routinely worked with small images, sometimes just one line, one samp, many bands

Reply:
Also, this call doesn't force the stop_time to be the last entry in the return. Should we add the endpoint arg to force it?

Reply:
Adding the endpoint arg doesn't force both endpoints when you pass 1 for the number of samples. I've changed this to always do at least 2.